src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx

```diff
@@ -0,0 +1,260 @@
+```  
+OR   
+```
+FROM PageView SELECT uniqueCount(session) WHERE appName in ('CustomerApp1', 'CustomerApp2')
+```  
+
+If you provide an API as a service, capture customer IDs, and it's relevant to your business, you can track the equivalent of users by tracking customers in transactions:    
+```
+FROM Transaction uniqueCount(customerId) WHERE appName = 'apiService'
+```
+
+  </Collapser>
+</CollapserGroup>
+
+The business KPIs above work on the assumption that you support your users by providing a front-end application. If you support your customers via API, it may be possible to fit the above KPIs to the Transaction entity type. Some organizations that provide APIs as a service use operational KPIs like the ones below to promote the quality of their offering.
+
+### Operational KPIs
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-error-operational-error-rate"
+    title="Error rate"
+  >
+The error rate is the ratio of errors to requests. 
+
+An error can be a 300+ HTTP response code, an unhandled exception, a mobile crash, or an event someone on your team configured to be an error. 
+
+**Goal:** Reduce the error rate across the applications you manage.
+
+**Best practices** 
+This is the main KPI you will use to track your progress against improving error tracking. Steps to improve error rates include filtering out inconsequential errors as well as resolving impactful errors.
+
+  </Collapser>
+
+  <Collapser
+    id="kpi-error-operational-error-mean-time"
+    title="Mean time to close errors"
+  >
+Mean time to close measures the time from when an alert notifies you of an issue until the issue is closed, just after it has been resolved.  This KPI is tracked as part of [Alert quality management](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide#key-perf-indicators)
+
+**Goal:** Reduce the mean time to close issues by reducing the error rate.
+
+**Best practices** 
+Build a strong error practice so that when an issue occurs, you will be able to more quickly detect:
+* Whether or not the issue is related to a spike in errors
+* Which error is responsible for the issue, if the issue is caused by an error
+
+The [Alert quality management guide](/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide) shows you how to track this KPI. 
+  </Collapser>
+</CollapserGroup>
+
+## Prerequisites [#prerequisites]
+
+### Required installation and configuration
+* Make sure your errors are being captured by our APM, browser monitoring, mobile monitoring, serverless monitoring, or OpenTelemetry solutions. 
+* Up-to-date [source maps](/docs/browser/new-relic-browser/browser-pro-features/upload-source-maps-api) for web applications
+* Up-to-date [symbolification](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting) for mobile applications 
```

Comment:
you had a docs search link here, but i'm curious if this link is better. mainly asking because our docs search isn't that great